front/src/applications/operationalStudies/views/Project.tsx

```diff
@@ -118,7 +118,7 @@ export default function Project() {
         };
         try {
           const filteredData = await postSearch(payload).unwrap();
-          let filteredStudies = [...filteredData] as SearchStudyResult[];
+          let filteredStudies = [...filteredData] as SearchResultItemStudy[];
```

Comment:
Same here, you can delete l120
```suggestion
          let filteredStudies = await postSearch(payload).unwrap() as SearchResultItemStudy[];
```